CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 2.2.0 - 2022-12-05
+### Added
+- Google OAuth support
+- Skeleton for thread
+- Make HTML body loading a single step experience
+- Refactor HTML body element ID to class
+- Give thread messages a minimum height
+- Show thread subject while loading thread messages
+- Local Draft Handling Backend
+- Search box for outbox and PI
+- Support for email field
+### Fixed
+- Advanced search does not scroll
+- Dark theme loading envelops and thread colour
+- Scrolling for long threads on narrow screens
+- Replace emptycontent div with EmptyContent vue component
+- Drop right border from navigation header
+- Saving provisioning and anti-spam settings
+- Don't decode content for preview twice
+- Broken encoding for outgoing messages
+- Add empty content for loading thread
+- Improve the size of padding bottom
+- Provisioning config not saving
+- Sending PGP messages as HTML
+- Copy to clipboard
+- Envelope and message previews showing PGP ciphertext
+- Double scrollbar in thread view
+- Alias provisioning: Skip alias when identical to account email
+- Improve the dropdown multiselect menu of default folders
+- XOAUTH2 auth via SMTP
+- Disable provisioned accounts if using passwordless authentication
```

Comment:
```suggestion
- Disable provisioned accounts if using passwordless authentication
- Fix dragover indicator being inherited to children
```

https://github.com/nextcloud/mail/pull/7728

Reply:
Depending on the answer to https://github.com/nextcloud/mail/pull/7727#issuecomment-1337252504.